pkgs/development/compilers/opensmalltalk-vm/default.nix

```diff
@@ -103,84 +92,102 @@ let
       '';
 
       meta = {
-        description = "The cross-platform virtual machine for Squeak, Pharo, Cuis, and Newspeak.";
-        mainProgram = scriptName;
+        description =
+          "The cross-platform virtual machine for Squeak, Pharo, Cuis, and Newspeak.";
+        mainProgram = finalAttrs.scriptName;
         homepage = "https://opensmalltalk.org/";
         license = with lib.licenses; [ mit ];
-        maintainers = with lib.maintainers; [ jakewaksbaum ];
-        platforms = [ stdenv.targetPlatform.system ];
+        maintainers = with lib.maintainers; [ jakewaksbaum ehmry ];
+        inherit platforms;
```

Comment:
Here `platforms` is set to `builtins.attrNames platforms` of the platforms define later at the VM.